docs/table.clone-locked-metatables.md

```diff
@@ -0,0 +1,73 @@
+# Feature name
+
+## Summary
+
+Allow `table.clone` to copy tables with locked metatables.
+
+## Motivation
+
+As proposed by the [`table.clone` RFC](function-table-clone.md), the function `table.clone` cannot create shallow copies of a table if it has a locked metatable.
+Due to this ugly limitation, it is extremely un-ergonomic to create shallow copies of arbitrary tables.
+The current behavior of using `table.clone` on a table with a locked metatable is a hard-fail approach where `table.clone` spits out an error instead of having a soft-fail approach wherein a clone is generated without a metatable.
+This hard-fail approach severely hinders the usefulness of `table.clone` due to in practice users having to re-implement the function in many scenarios.
```

Comment:
Since there are many scenarios, can one or two examples be included in the motivation?

Examples can also note how dropping the metatable is handled in them (given that it will be dropped only if it's protected).